Doc/library/http.server.rst

```diff
@@ -51,9 +51,43 @@ handler.  Code to create and run the server looks like this::
    .. versionadded:: 3.7
 
 
-The :class:`HTTPServer` and :class:`ThreadingHTTPServer` must be given
-a *RequestHandlerClass* on instantiation, of which this module
-provides three different variants:
+.. class:: HTTPSServer(server_address, RequestHandlerClass,\
+                       bind_and_activate=True, *, certfile, keyfile=None,\
+                       password=None, alpn_protocols=None)
+
+   Subclass of :class:`HTTPServer` with a wrapped socket using the :mod:`ssl` module.
+   If the :mod:`ssl` module is not available, instantiating an :class:`!HTTPSServer`
```

Comment:
```suggestion
   If the :mod:`ssl` module is not available, instantiating a :class:`!HTTPSServer`
```

I think I was wrong before. It should *a* HTTP because it's a Hypertext and not an Hypertext. My bad.